docs/maintaining.md

```diff
@@ -160,22 +190,39 @@ curl -X PUT --header 'Content-Type: application/json' -d '
 
 ### Resetting your GraphDB instance
 
-If you previously set up a Neurobagel node on your machine but want to reset your graph database to start again _from scratch_, 
-the most foolproof way would be to start with a clean GraphDB configuration to avoid conflicts with any previously created credentials or databases.
+Each Neurobagel node has its own GraphDB instance, which is used to store the graph data for the node.
+If you want to reset your graph database and start again from scratch,
+follow these steps:
+
+1. Ensure that your Neurobagel node is not running (i.e., shut down the Docker containers for the node).
+
+    ```bash
+    docker compose --profile full_stack down
+    ```
+
+    If you are not using the `full_stack` profile, replace `full_stack` with the name of the profile you are using.
+
+2. Delete the docker volume that contains the GraphDB data for your node.
+
+    ```bash
+    docker volume rm neurobagel_graphdb_data
+    ```
+
+    replace `neurobagel_graphdb_data` with the name of the volume that was created for your node.
+    It is usually named `<project_name>_graphdb_data`
+    where `<project_name>` is the name of the directory where you have stored the `docker-compose.yml` file for your Neurobagel node.
+
+3. Start your Neurobagel node again.
+
+    ```bash
+    docker compose --profile full_stack up -d
+    ```
```

Comment:
```suggestion
    ```

    If you are not using the `full_stack` profile, replace `full_stack` with the name of the profile you are using.
```